service/internal/logger/logger.go

```diff
@@ -39,22 +64,42 @@ func NewLogger(config Config) (*Logger, error) {
 	switch config.Type {
 	case "json":
 		j := slog.NewJSONHandler(w, &slog.HandlerOptions{
-			Level: level,
+			Level:       level,
+			ReplaceAttr: customReplaceAttributes,
 		})
 		logger = slog.New(j)
 	case "text":
 		t := slog.NewTextHandler(w, &slog.HandlerOptions{
-			Level: level,
+			Level:       level,
+			ReplaceAttr: customReplaceAttributes,
 		})
 		logger = slog.New(t)
 	default:
 		return nil, fmt.Errorf("invalid logger type: %s", config.Type)
 	}
+
+	// Audit logger will always log at the AUDIT level and be JSON formatted
+	auditLoggerHandler := slog.NewJSONHandler(w, &slog.HandlerOptions{
+		Level:       LevelAudit,
+		ReplaceAttr: customReplaceAttributes,
+	})
+
+	auditLoggerBase := slog.New(auditLoggerHandler)
+	auditLogger := CreateAuditLogger(*auditLoggerBase)
+
 	return &Logger{
 		Logger: logger,
+		Audit:  auditLogger,
```

Comment:
is there a way to disable audit logging? (or do we want that?)

Reply:
So for this, I originally had the level set to in between error and warning, and then @strantalis I believe suggested it be higher than error and not something we can turn off

The way it's designed now we could probably pass a config option to turn off audit logging, and if that is false all the `logger.Audit` sub logger could be turned to level "off" instead of whatever it is set at

@strantalis what do you think, should we do that?